keps/prod-readiness/sig-cli/2227.yaml

```diff
@@ -1,3 +1,5 @@
 kep-number: 2227
 alpha:
   approver: "@deads2k"
+beta:
+  approver: "@deads2k"
```

Comment:
You'll need to fill in both beta (retroactively) and stable.

Reply:
Updated.
@deads2k